src/assets/json/statement.json

```diff
@@ -0,0 +1,4 @@
+{
+  "title": "OoT has reached 100%!",
+  "content": "<p>It's been a wild ride. We've been able to create C code that, when compiled, reproduces the original game. We call this \"matching\" decompilation. On 26 November 2021, Fig02, who is a notable community member as well as a project lead, matched the last-remaining function in the first version of Ocarina of Time. This means that all compiled code in the game has been turned into human-readable C code. We thought for a time that we may never be able to match every function completely, so this is an incredibly exciting accomplishment. Dozens of people helped work on this project, and together we were able to achieve something amazing</p><p>There's still a lot of remaining work for the project, however. In no particular order:</p><ul><li>Documentation - Lots of code is not well understood, still, so we need to reverse-engineer things and document them.</li><li>Re-organization -We plan to reorganize the code and definitions in the project.</li><li>Re-naming - many names of things are not ideal for finding the code for a particular part of the game, so we plan to standardize and rename many systems and components.</li><li>Other versions - We have been working on decompiling the Master Quest Debug version of the game. However, Ocarina of Time has 17 other versions, which we plan to also decompile and support in the project.</li><li>Asset support - We plan to continue to improve support of asset-handling so that viewing / modifying assets on modern computers is easier to do.</li><li>Other games - We have many other projects in progress: Majora's Mask, The Minish Cap, Twilight Princess, and Breath of the Wild all need help as well to reach 100%!</li></ul><p style=\"float: right;\">ZeldaRET project leads</p>"
```

Comment:
```suggestion
  "content": "<p>It's been a wild ride. We've been able to create C code that, when compiled, reproduces the original game. We call this \"matching\" decompilation. On 26 November 2021, Fig02, who is a notable community member as well as a project lead, matched the last-remaining function in the first version of Ocarina of Time. This means that all compiled code in the game has been turned into human-readable C code. We thought for a time that we may never be able to match every function completely, so this is an incredibly exciting accomplishment. Dozens of people helped work on this project, and together we were able to achieve something amazing</p><p>There's still a lot of remaining work for the project, however. In no particular order:</p><ul><li>Documentation - Lots of code is not well understood, still, so we need to reverse-engineer things and document them.</li><li>Re-organization -We plan to reorganize the code and definitions in the project.</li><li>Re-naming - many names of things are not ideal for finding the code for a particular part of the game, so we plan to standardize and rename many systems and components.</li><li>Other versions - We have been working on decompiling the Master Quest Debug version of the game. However, Ocarina of Time has 17 other versions, which we plan to also decompile and support in the project.</li><li>Asset support - We plan to continue to improve support of asset-handling so that viewing / modifying assets on modern computers is easier to do.</li><li>Other games - We have many other projects in progress: Majora's Mask, The Minish Cap, Twilight Princess, and Breath of the Wild all need help as well to reach 100%!</li></ul><p style=\"float: right;\">— ZeldaRET project leads</p>"
```